src/account/account.actions.js

```diff
@@ -9,15 +9,15 @@ const accountActions = Action.createActionsFromNames([
     'setPassword',
 ]);
 
-accountActions.setPassword.subscribe(({ data: password, complete, error }) => {
+accountActions.setPassword.subscribe(({ data: [password, onSuccess], complete, error }) => {
```

Comment:
So `accountActions` just pass `arguments` as `data` to the subscribtion handlers?